src/main/java/uk/gov/hmcts/dts/fact/model/Court2.java

```diff
@@ -53,34 +70,31 @@ public Court2(uk.gov.hmcts.dts.fact.entity.Court2 courtEntity) {
         this.magistratesLocationCode = courtEntity.getMagistrateCode();
         this.slug = courtEntity.getSlug();
         this.courtTypes = courtEntity.getCourtTypes().stream().map(CourtType::getName).sorted().collect(toList());
-        this.address = this.mapAddress(courtEntity);
+        this.address = this.mapAddress(courtEntity.getAddresses());
         this.areasOfLaw = courtEntity.getAreasOfLaw().stream().map(AreaOfLaw::new).collect(toList());
         this.displayed = courtEntity.getDisplayed();
         this.hideAols = courtEntity.getHideAols();
-        this.dxNumber = this.getDxNumber(courtEntity);
+        this.dxNumber = this.getDxNumber(courtEntity.getContacts());
         this.distance = BigDecimal.valueOf(courtEntity.getDistance()).setScale(2, RoundingMode.HALF_UP);
     }
 
-    private String getDxNumber(uk.gov.hmcts.dts.fact.entity.Court2 courtEntity) {
-        return courtEntity
-            .getContacts()
+    private String getDxNumber(final List<Contact> contacts) {
+        return contacts
             .stream()
             .filter(c -> "DX".equals(c.getName()))
             .map(Contact::getNumber)
             .findFirst()
             .orElse(null);
     }
 
-    private CourtAddress2 mapAddress(uk.gov.hmcts.dts.fact.entity.Court2 courtEntity) {
-        return courtEntity
-            .getAddresses()
+    private CourtAddress2 mapAddress(List<CourtAddress> courtAddresses) {
```

Comment:
Is the address different as well?

Reply:
i believe it splits the address into an array. currently the string variable has \n line in them.

Reply:
exactly